Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1beta3/conversion.go

```diff
@@ -126,7 +126,7 @@ func addConversionFuncs() {
 	err = api.Scheme.AddFieldLabelConversionFunc("v1beta3", "Namespace",
 		func(label, value string) (string, string, error) {
 			switch label {
-			case "status.phase":
+			case "name", "metadata.name", "status.phase":
```

Comment:
yeah, field selectors are only used for querying/filtering... you can't query/filter v1beta3 any more